dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/MemoryInformation.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018 IBM Corporation and others.
+ * Copyright (c) 2021 IBM Corporation and others.
```

Comment:
Should list both dates
```suggestion
 * Copyright (c) 2018, 2021 IBM Corporation and others.
```
Don't really need a new PB to fix that though

Reply:
Done, thanks